modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -86,7 +87,7 @@ public class CatalogUtils {
      *
      * <p>SQL`16 part 2 section 6.1 syntax rule 38
      */
-    public static final int MAX_TIME_PRECISION = 9;
+    public static final int MAX_TIME_PRECISION = NativeTypes.MAX_TIME_PRECISION;
```

Comment:
Let's try to move constants to IgniteTypeSystem or NativeTypes.

Reply:
I filed an issue for this https://issues.apache.org/jira/browse/IGNITE-20619